nodewatcher/modules/importer/nw2/management/commands/import_nw2.py

```diff
@@ -277,7 +277,7 @@ def import_pools(self, data):
             pool_mdl = pool_models.IpPool(
                 family='ipv4',
                 network=pool['network'],
-                prefix_length=pool['cidr'],
+                prefix_length=pool['ip_space'],
```

Comment:
These changes in `import_nw2.py` seem incorrect - was this a too broad search and replace?